protocol.go

```diff
@@ -0,0 +1,112 @@
+package tarantool
+
+import (
+	"context"
+	"fmt"
+)
+
+type ProtocolVersion uint64
+type ProtocolFeature uint64
+
+type ProtocolInfo struct {
+	Version  ProtocolVersion
+	Features []ProtocolFeature
+}
```

Comment:
Let's add a description to types/fields because this is a public API.

Reply:
Added